setup.cfg

```diff
@@ -127,6 +132,8 @@ all =
     kubernetes_asyncio
     paramiko
     impyla
+    openlineage-airflow==0.6.2
```

Comment:
Pinning a library will cause dependency conflicts when users use our library with other libraries.

So we should not pin it to a specific version unless we have a really strong reason to do it

Reply:
Yes, 0.7.0 release was unstable; the package was not getting installed correctly due to some release depedency issue on OpenLineage release process and packages <0.6.2 had some missing fixes, so freezed it to 0.6.2 which tested to run fine

Reply:
0.8.1 is out now though - https://pypi.org/project/openlineage-airflow/

Reply:
added a response below. https://github.com/astronomer/astronomer-providers/pull/290#issuecomment-1121191492